### ISSUE TITLE
[Bug]: kicks me out of server after opening a chest filled with printing forms

### ISSUE THREAD

**Issue by nk34player** [AllTheMods/ATM-10#1292]
### Possible Fixes

Yes

### Modpack Version

1.25

### What happened?

If i just have some printing forms in a chest once i use them, they disappear and keeps crashing my game and kicking me out here is the error
io.netty.handler.codec.EncoderException: Failed to encode packet 'clientbound/minecraft:container_set_content'

https://mclo.gs/p7Fqsea

here is the log of the crash

### Developer reports

* https://github.com/TeamDman/SuperFactoryManager/issues/209
* https://github.com/TeamDman/SuperFactoryManager/issues/213

**Comment by TeamDman** [comment]
Very strange, I don't doubt this is caused by sfm its just not showing what part of the mod is responsible because all the stack traces are from vanilla methods 😭 

Will investigate

Are you able to reproduce in a new world from scratch? step by step Instructions would help

**Comment by nk34player** [comment]
Yes, using the press once it works after that shift clicking to take the printing form also kicks me out with the same error, then the error keep present even when opening chests filled with printing forms

**Comment by TheBedrockMaster** [comment]
@TeamDman Was there any update on this?

**Comment by TeamDman** [comment]
Am aware of reported problems with the printing press and have a note to reproduce and fix hopefully for the next release which I am actively working on

**Comment by trunip190** [comment]
Here's a little more info on triggering it and what doesn't:
using pipes to move the press from the printer to a barrel doesn't trigger.
opening the barrel does then trigger it.
moving press from barrel back to printer does not trigger it.

not sure why, but some times (maybe first usage?) i can print a book and then immediately remove the press without triggering.
breaking printer by hand triggers.
breaking printer by machine also triggers.

the error i get is "failed to encode item", so my best guess is that when the item is created (by drop or removing from the printer) it fails to correctly create the item and then fails item validation. 

I hope this helps

**Comment by TeamDman** [comment]
Should be fixed in the latest build SFM v4.20.0
